src/aliceVision/matching/RegionsMatcher.cpp

```diff
@@ -10,172 +10,190 @@
 #include "aliceVision/matching/ArrayMatcher_bruteForce.hpp"
 #include "aliceVision/matching/ArrayMatcher_kdtreeFlann.hpp"
 #include "aliceVision/matching/ArrayMatcher_cascadeHashing.hpp"
+#include "aliceVision/matching/ArrayMatcher_hnswlib.hpp"
 
```

Comment:
Please reformat le code using spaces instead of tabs

Reply:
I just pushed the reformatting changes, please review and let me know if I missed a spot to reformat :)